ai-decision-services/ai-ad-dpp-engine/prepare-config-data-and-app/prepare-config-data-and-app.md

```diff
@@ -0,0 +1,217 @@
+                "steps": [
+                    {
+                        "stepName": "format_timestamp",
+                        "args": {}
+                    }
+                ]
+            }
+        }
+    ],
+    "stagingDestination": {
+        "combinedResult":"S1",
+        "type": "object-storage",
+        "namespace":"<your-namespace>",
+        "bucket": "staging-bucket",
+        "folder": "training_processed_data"
+    },
+    "outputDestination": {
+        "type": "object-storage",
+        "namespace":"ax3dvjxgkemg",
+        "bucket": "output-bucket",
+        "objectName": "model_info.json",
+    },
+    "serviceApiConfiguration": {
+        "anomalyDetection": {
+            "serviceEndpoint": null,
+            "profileName": "DEFAULT",
+            "projectId": "<ad-project-ocid>",
+            "compartmentId": "<your-compartment-ocid>"
+        }
+    }
+}
+```
+
+**Inferencing config** 
+
+```java
+{
+    "inputSources": [
+        {
+            "dataframeName": "S1",
+            "type": "object-storage",
+            "namespace":"<your-namespace>",
+            "bucket":"inferencing-data-bucket",
+            "objectName": "TestData.csv"
+        }
+    ],
+    "phaseInfo": {
+        "connector": {
+            "namespace":"<your-namespace>",
+            "bucket":"output-bucket",
+            "objectName": "metadata.json"
+        }
+    },
+    "processingSteps": [
+        {
+            "stepType": "singleDataFrameProcessing",
+            "configurations": {
+                "dataframeName": "S1",
+                "steps": [
+                    {
+                        "stepName": "format_timestamp",
+                        "args": {}
+                    }
+                ]
+            }
+        }
+    ],
+    "stagingDestination": {
+        "combinedResult":"S1",
+        "type": "object-storage",
+        "namespace":"<your-namespace>",
+        "bucket": "staging-bucket",
+        "folder": "training_processed_data"
+    },
+    "outputDestination": {
+        "type": "object-storage",
+        "namespace":"<your-namespace>",
+        "bucket": "output-bucket",
+        "objectName": "model_info.json",
+    },
+    "serviceApiConfiguration": {
+        "anomalyDetection": {
+            "serviceEndpoint": null,
+            "profileName": "DEFAULT",
+            "projectId": "<ad-project-id>",
+            "compartmentId": "<your-compartment-ocid>",
+            "modelId": "<>"
+        }
+    }
+}
+```
+
+  
+
+## Driver code
+
+This is an executable provided by us. Here is our latest driver code that can conduct data preprocessing, training and inferencing: [link](https://github.com/bug-catcher/oci-data-science-ai-samples/blob/master/ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/df_driver.py).
+
+## Training/Inferencing datasets
+
+ Here we attached a usable set of training and testing data in csv and parquet. 
```

Comment:
Reword - Attached are a set of training and testing (/inference) data files in CSV and Parquet formats.